tools/serverpod_cli/lib/src/language_server/language_server.dart

```diff
@@ -84,11 +84,6 @@ Future<void> runLanguageServer() async {
       ),
     );
 
-    statefulAnalyzer.validateProtocol(
```

Comment:
Question: If this was removed, should we also remove the call inside `.onDidChangeTextDocument(...)`?

Reply:
We cannot do that as the statefulAnalyzer has no other method to modify a yaml file. Running this method is the only way todo that. 

With that said #1098 refactors this.